packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -102,8 +102,11 @@ const settingsConfig = {
   },
   featureFlags: {
     keyStretchV2: config.get('featureFlags.keyStretchV2'),
-    recoveryCodeSetupOnSyncSignIn: config.get(
```

Comment:
Was this intentionally removed?

Reply:
Mght be the cause of fxa-content-server failing to start up for functional tests on ci

Reply:
Ha oops no, thanks. Bad cut and pastes.